nb/api/Assembly.md

```diff
@@ -1,20 +1,29 @@
-Assembly(...shapes)
+### Assembly()
+Parameter|Default|Type
+---|---|---
+...shapes||The shapes to assemble.
 
 Produces a disjoint assembly of shapes.
```

Comment:
Is it safe to assume that our reader understands what disjoint means? I didn't know that meaning before. That being said I don't know how to clarify it more without just defining 'disjoint' and they can easily look up a definition of disjoint

Reply:
Never mind, I think the example clarifies it well 👌